src/lib/__tests__/cache.test.ts

```diff
@@ -40,39 +43,41 @@ describe("Cache with compression enabled", () => {
       beforeEach(async () => await cache.set("get_foo", { bar: "baz" }))
 
       it("deletes the data", async () => {
-        await cache.delete("get_foo")
         expect.assertions(1)
-        try {
-          await cache.get("get_foo")
-        } catch (e) {
-          expect(e.message).toEqual("[Cache#get] Cache miss")
-        }
+
+        await cache.delete("get_foo")
+
+        await expect(cache.get("get_foo")).rejects.toThrow(
+          "[Cache#get] Cache miss"
+        )
       })
     })
 
     describe("#set", () => {
       describe("with a plain Object", () => {
-        it("sets the cache and includes a timestamp", async (done) => {
+        it("sets the cache and includes a timestamp", async () => {
+          expect.assertions(2)
+
           await cache.set("set_foo", { bar: "baz" })
 
-          client.get(cacheKey("set_foo"), (_err, data) => {
+          await client.get(cacheKey("set_foo"), (_err, data) => {
```

Comment:
Does the eslint error instruct to use `await` if this is forgotten?

Reply:
Oh, I may have just misunderstood this one. I guess it's just the callback not returning a promise.

In general eslint will error on thenables that aren't returned with `Promise should be returned to test its fulfillment or rejectioneslintjest/valid-expect-in-promise` but yeah this isn't one of those.